### DISCUSSION TITLE
Sat endpoints can't call content directly?

### DISCUSSION THREAD

**Discussion by cliobitcoinbank** [ordinals/ord#3053] [General]
I was working on a recursive collection and wanted to use the /r/sat/2505000009000/at/-1 endpoint so the ordinals would always point to the latest re-inscription but it pulls the inscription id instead of the content itself so <script s=1000 src="/r/sat/2505000009000/at/-1"> doesnt work because it can't call the content it just returns {"id":"08881555ae7f521bdabfd8612157f6bec481cc458047cc38b32b54a8c8ab85eei0"} which forces me to make a second recursive call in order to get to the content i want.  Having to run 2 recursions is bloating my ordinals from 90 bytes to 190bytes!! Thats huge!!  Am i missing something obvious or could we get an endpoint that calls the latest reinscription on a sat like /r/sat/2505000009000/new or something like that so my collection pieces can remain very very smol by avoiding having to do 2 recursions?  Thank you.

**Comment by raphjaph** [comment]
Yes, that is by design for caching purposed. You will have to get the id and then call the `/content/<ID>` endpoint.

**Comment by cliobitcoinbank** [comment]
Understood, ended up making a middleware script ordinal I call using a static /content/ first and that middleware resolves the  /r/sat/2505000009000/at/-1 endpoint for me and returns it's content, its a pain but works really well.  Just adding that note here before closing in case someone has the same issue and needs the solution of using a /sat/ endpoint resolver between their ordinal and the content at the sat endpoint they want.
